{"dataset": {"name": "openml__cnae-9__9981", "cat_idx": [], "cat_dims": [], "target_type": "classification", "num_classes": 9, "num_features": 856, "num_instances": 1080, "split_source": "openml"}, "scaler": "Quantile", "model": {"name": "TabFlexModel", "params": {}, "args": {"model_name": "TabFlexModel", "batch_size": 128, "scale_numerical_features": "Quantile", "val_batch_size": 256, "objective": "classification", "gpu_ids": [0], "use_gpu": false, "epochs": 500, "data_parallel": true, "early_stopping_rounds": 20, "dataset": "openml__cnae-9__9981", "cat_idx": [], "num_features": 856, "subset_features": -1, "subset_rows": -1, "subset_features_method": "random", "subset_rows_method": "random", "cat_dims": [], "num_classes": 9, "logging_period": 100}}, "experiemnt_args": {"experiment_config": "TabZilla/new_experiment_config.yml", "output_dir": "./results/", "write_predictions": false, "use_gpu": false, "gpu_ids": [0], "data_parallel": true, "n_random_trials": 30, "hparam_seed": 0, "n_opt_trials": 0, "batch_size": 128, "val_batch_size": 256, "scale_numerical_features": "Quantile", "early_stopping_rounds": 20, "epochs": 500, "logging_period": 100, "experiment_time_limit": 36000, "trial_time_limit": 7200, "subset_rows": -1, "subset_features": -1, "subset_rows_method": "random", "subset_features_method": "random", "subset_random_seed": 0}, "hparam_source": "default", "trial_number": 0, "exception": "Traceback (most recent call last):\n  File \"/datadrive/tabzilla/TabZilla/tabzilla_experiment.py\", line 159, in __call__\n    result = cross_validation(\n  File \"/datadrive/tabzilla/TabZilla/tabzilla_utils.py\", line 274, in cross_validation\n    loss_history, val_loss_history = curr_model.fit(\n  File \"/datadrive/tabzilla/TabZilla/models/tabflex.py\", line 22, in fit\n    self.model.fit(X, y, overwrite_warning=True)\n  File \"/datadrive/tabpfn_training_amueller/mothernet/prediction/tabpfn.py\", line 217, in fit\n    model, c, results_file = load_model_workflow(\n  File \"/datadrive/tabpfn_training_amueller/mothernet/prediction/tabpfn.py\", line 69, in load_model_workflow\n    raise Exception('No checkpoint found at '+str(model_path))\nException: No checkpoint found at /datadrive/tabpfn_training_amueller/mothernet/models_diff/ssm_tabpfn_b4_maxnumclasses100_modellinear_attention_numfeatures1000_n1024_validdatanew_warm_08_23_2024_19_25_40_epoch_100.cpkt\n", "timers": {}, "scorers": {}}